### ISSUE TITLE
DataGrid paging control accessibility

### ISSUE THREAD

**Issue by haydnkane** [Megabit/Blazorise#4890]
**Describe the bug**
There are two accessibility issues with the paging control:

- There is a div inside the ul which fails WCAG 1.3.1
- 'items per page' select does not a have a label/aria-label which fails WCAG 4.1.2

**To Reproduce**

1. Using the Accessibility Insights for Web extension for Edge or Chrome and run FastPass against a page with a datagrid that has the paging controls

**Expected behavior**
No reported WCAG accessibility issues for the paging control

**Screenshots**
![Screenshot 2023-07-20 142025-2](https://github.com/Megabit/Blazorise/assets/21358297/4a15232a-832e-4324-a724-92b939db5bcc)
![Screenshot 2023-07-20 141952](https://github.com/Megabit/Blazorise/assets/21358297/60054cf9-1d19-4236-9722-418261a1ec8e)

**Comment by stsrki** [comment]
First of all, thank you for letting us know about the Accessibility Insights for Web extension. We were not aware of it.

Regarding the accessibility issues, we already have it on the backlog, #3705. The plan is to start working on it sometimes after 1.3 release.
